MC/bin/o2dpg_sim_workflow.py

```diff
@@ -744,7 +745,7 @@ def createRestDigiTask(name, det='ALLSMALLER'):
                                                     'TPCGasParam',
                                                     'ITSCATrackerParam',
                                                     'MFTClustererParam'],{"ITSClustererParam.dictFilePath":"../"}) \
-                              + " --track-sources " + anchorConfig.get("o2-tof-matcher-workflow-options",{}).get("track-sources","all")
+                              + " --track-sources " + anchorConfig.get("o2-tof-matcher-workflow-options",{}).get("track-sources","ITS-TPC,TPC")
```

Comment:
why this? Shouldn't the default be all?

Reply:
I will restore to "all" (I put this as a temporary fix not needed anymore, a long story)

Reply:
done